### PR TITLE
docs: add README zh translation and fix some bad links

### PR DESCRIPTION
**Pull request by suyanhanx**
Signed-off-by: suyanhanx <suyanhanx@gmail.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 _output/
 _artifacts/
 lima.REJECTED.yaml
+.idea/
```
Comment (AkihiroSuda) [inline .gitignore:4]: This should be in your local gitignore

**Comment by AkihiroSuda** [comment]
Also please consider squashing commits

**Comment by suyanhanx** [comment]
> Also please consider squashing commits

Yes. Before merging, I'll do this.

File: .gitignore (hunk 1 above)
Comment (suyanhanx) [reply .gitignore]: Done.

**Comment by AkihiroSuda** [comment]
Could you update the DCO sign in the git commit message too?
https://github.com/lima-vm/lima/pull/1296/commits

**Comment by suyanhanx** [comment]
> Could you update the DCO sign in the git commit message too? https://github.com/lima-vm/lima/pull/1296/commits

Done.

**Review by AkihiroSuda — APPROVED** [review]
Thanks

**Pull request merged**